package.json

```diff
@@ -29,15 +29,15 @@
     "@angular/animations": "6.1.7",
     "@angular/common": "6.1.7",
     "@angular/compiler": "6.1.7",
-    "@angular/core": "6.1.7",
+    "@angular/core": "^6.1.7",
```

Comment:
why ^ just for core?

Reply:
Well spotted, this gets added by ng update @angular/core.  Do you think we should use "^" on all angular code?

Reply:
I avoid it in our project. I don't want things slipping in w/o  me being aware.